gke-disk-image-builder/cli/main.go

```diff
@@ -65,6 +66,9 @@ func main() {
 	flag.Var(&imageLabels, "image-labels", "labels tagged to the disk image. This flag can be specified multiple times. The accepted format is `--image-labels=key=val`.")
 	flag.Var(&containerImages, "container-image", "container image to include in the disk image. This flag can be specified multiple times")
 
+	instanceDiskSizeGb := newInstanceSizeGBFlag(diskSizeGb)
+	flag.Var(instanceDiskSizeGb, "instance-disk-size-gb", "disk size of the GCE instance used for disk image creation. If not set, defaults to the value of --disk-size-gb.")
```

Comment:
Can help explain why we need the struct intInstanceSizeGBFlag?

This line and the struct can be replaced by
```
instanceDiskSizeGb := flag.Int64("instance-disk-size-gb", *diskSizeGb, "disk size of the GCE instance used for disk image creation. If not set, defaults to the value of --disk-size-gb.")
```

Reply:
Actually I'm not sure right now. I remember that it was my first guess to use diskSizeGb — it's simple and didn't require additional code, but then it didn't work or I was unsatisfied with what I saw in `--help` menu